packages/core/src/Accordion/Accordion.test.tsx

```diff
@@ -27,7 +27,7 @@ const Controlled = () => {
     <HvAccordion
       label="Analytics"
       expanded={open}
-      onChange={() => setOpen(!open)}
+      onChange={(event, value) => setOpen(value)}
```

Comment:
Isn't the point of controlled components that we use our own state, instead of the provided by the component? 🤔